pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/regionalcomputeforwardingrule/_http.log

```diff
@@ -776,10 +715,12 @@ X-Xss-Protection: 0
 
 ---
 
-GET https://compute.googleapis.com/compute/v1/projects/${projectId}/global/operations/${operationID}?alt=json&prettyPrint=false
```

Comment:
I had thought this looked like we weren't checking the LRO on delete, but ... we are.

Reply:
Updated the log.
I think that's because I made a mistake in mockgcp/mockcompute/regionalforwardingrulev1.go: Should use return s.startRegionalLRO not return s.startGlobalLRO. I updated the mock as well.